context/app/static/js/shared-styles/tiles/Tile/Tile.tsx

```diff
@@ -33,7 +33,11 @@ function Tile({
     </StyledPaper>
   );
   if (href) {
-    return <a href={href}>{tile}</a>;
+    return (
+      <a href={href} aria-label={href}>
+        {tile}
```

Comment:
Not sure if we want to parse the hrefs in this case by extracting keywords from them. But it would need some kind of mapping, easy for organs, difficult for datasets page with long uuids that are not meaningful

Reply:
Ditto here - I think constructing a descriptive text would be more straightforward if we were to define that logic from the parent element that uses these tiles.